releases/cloud/point-2024-22-10-release-note.md

```diff
@@ -0,0 +1,30 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.5.67.1
+---
+
+# October 22, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+## Improvements & Bug Fixes
+
+* **New column added to the Settings for Custom Metadata.**
+  * **The Editing Allowed column** provides information on whether this metadata is enabled for owners to review. 
```

Comment:
osim kroz Metadata Review owneri mogu i generalno kroz Point mijenjati metadatu ako je ova opcija enabled, tako da ne bih samo Metadata Review spominjao nego općenito da je to opcija koja ownerima omogućava management custom metadate
npr.
..on whether owners can manage this metadata through Point overview reports and Metadata Review